test/smoke/test/index.js

```diff
@@ -18,21 +18,22 @@ const opts = minimist(args, {
 
 const options = {
 	useColors: true,
-	timeout: 60000,
+	//{{SQL CARBON EDIT}}
+	timeout: 60000 * 2,
+	//{{END}}
 	slow: 30000,
 	grep: opts['f']
 };
-
-if (process.env.BUILD_ARTIFACTSTAGINGDIRECTORY) {
-	options.reporter = 'mocha-multi-reporters';
-	options.reporterOptions = {
-		reporterEnabled: 'spec, mocha-junit-reporter',
-		mochaJunitReporterReporterOptions: {
-			testsuitesTitle: `${suite} ${process.platform}`,
-			mochaFile: path.join(process.env.BUILD_ARTIFACTSTAGINGDIRECTORY, `test-results/${process.platform}-${suite.toLowerCase().replace(/[^\w]/g, '-')}-results.xml`)
-		}
-	};
-}
+//{{SQL CARBON EDIT}}
```

Comment:
the original version of this is a special case for running in VSTS. Why don't we want to keep it here?

Reply:
yes, i am working on adding it back